public/v2/apps/etherpad.json

```diff
@@ -0,0 +1,77 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://github.com/ether/etherpad-lite/blob/develop/doc/docker.md ",
+    "displayName": "Etherpad",
+    "description": "Etherpad is a real-time collaborative editor for the web",
+    "dockerCompose": {
+        "version": "3.8",
+        "services": {
+            "$$cap_appname": {
+            "image": "etherpad/etherpad:$$cap_etherpad_version",
+            "ports": [
+                "9001:9001"
+            ],
+            "containerHttpPort": "9001",
+            "depends_on": [
+                "$$cap_appname-db"
+            ],
+            "environment": {
+                "NODE_ENV": "$$cap_env",

```

Comment:
I don't think there is benefit from letting the end user set this value, we can hardcode it to `production`

Reply:
done